### DISCUSSION TITLE
32nd open maintainers call

### DISCUSSION THREAD

**Discussion by mxinden** [libp2p/rust-libp2p#4905] [General]
Hi everyone,

The rust-libp2p maintainers will meet tomorrow (Tuesday) 11:00 UTC. See [libp2p public events calendar](https://calendar.google.com/calendar/embed?src=libp2p.io_0q9682i3te7eanhe9q7ae1c58g%40group.calendar.google.com).

https://meet.jit.si/rust-libp2p-open-maintainers-call

Everyone is welcome to join, listen along, propose topics and join the discussions.

Please post below in case you already want to add an agenda item or list it in [our notes document](https://hackmd.io/xoHKlFuER0OVj1DGDuFROw).

See you all tomorrow!

**Comment by thomaseizinger** [comment]
cc @eserilev In case you want to chat about or have any questions in regards to https://github.com/libp2p/rust-libp2p/issues/4457.

**Reply by eserilev** [reply]
thanks for the heads up! unfortunately I'm in the PST timezone at the moment so I wont be able to make it (it'll be 3am my time). I should have a draft PR open shortly w/ some questions regarding #4457

**Reply by thomaseizinger** [reply]
Okay yeah that is understandable 😁

PST is a hard one to accommodate for with me being in AEDT and Max in Europe.
